gitlab/client.py

```diff
@@ -818,7 +825,28 @@ def http_list(
 
         if page or as_list is True:
             # pagination requested, we return a list
-            return list(GitlabList(self, url, query_data, get_next=False, **kwargs))
+            gl_list = GitlabList(self, url, query_data, get_next=False, **kwargs)
+            items = list(gl_list)
+            if page is None and get_all is None and gl_list.per_page is not None:
+                if len(items) >= gl_list.per_page and (
+                    gl_list.total is None or len(items) < gl_list.total
+                ):
+                    total_items = "10,000+" if gl_list.total is None else gl_list.total
```

Comment:
10k is specific only to the (current) GitLab.com deployment, might be better to only provide the numbers we really know. IIRC it might be even different depending on the endpoint (e.g. commit list).

Reply:
Okay. I changed it to `many`